### PR TITLE
Copy over platform README and add PR template

### PR DESCRIPTION
**Pull request by jwilander**
To be used temporarily until we have a custom readme for the webapp. Also added a PR template.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+[A brief description of what this pull request does.]
+
+#### Ticket Link
+[Please link the GitHub issue or Jira ticket this PR addresses.]
+
+#### Checklist
+[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
+- [ ] Added or updated unit tests (required for all new features)
+- [ ] Has UI changes
+- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates
```
Comment (enahum) [inline PULL_REQUEST_TEMPLATE.md]: can you add PR's in server and redux?

Comment (jwilander) [reply PULL_REQUEST_TEMPLATE.md]: Yeah good idea

File: README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Mattermost
+
+Mattermost is an open source, self-hosted Slack-alternative from [https://mattermost.org](https://mattermost.org).
+
+It's written in Golang and React and runs as a single Linux binary with MySQL or Postgres. Every month on the 16th [a new compiled version is released under an MIT license](https://www.mattermost.org/download/).
+
```
Comment (jasonblais) [inline README.md]: 0/5, propose a note along the lines of 

```
This project hosts the webapp client code. The server code is hosted at [/mattermost-server](https://github.com/mattermost/mattermost-server).
```

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 dist
 .yarninstall
 node_modules
+*.swp
```
Comment (sudheerDev) [inline .gitignore]: Even if not in this PR - Probably `yarn-error.log`as well

**Pull request merged**